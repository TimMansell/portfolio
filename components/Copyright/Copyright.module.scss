@import "styles/scssvars";
@import "styles/utilities/pxtorem";

.copyright {
  color: var(--secondary-text-colour);
  text-align: center;
  font-size: pxtorem(14);
}
