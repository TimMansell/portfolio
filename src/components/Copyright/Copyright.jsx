import format from 'date-fns/format';

import styles from './Copyright.module.scss';

export const Copyright = () => {
  return (
    <p className={styles.copyright}>
      &copy; {format(new Date(), 'yyyy')}
      <br />
      Tim Mansell
    </p>
  );
};

export default Copyright;
