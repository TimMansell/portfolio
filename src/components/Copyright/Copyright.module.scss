@import 'utilities/pxtorem';

.copyright {
  color: var(--secondary-text-colour);
  text-align: center;
  font-size: pxtorem(14);
}
