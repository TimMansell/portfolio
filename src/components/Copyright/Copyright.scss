@import 'variables';

.copyright {
  color: $secondary-text-colour;
  text-align: center;
  font-size: 14px;
}
d