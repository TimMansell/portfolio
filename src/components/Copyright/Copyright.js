import React from 'react';
import format from 'date-fns/format';

import './Copyright.scss';

export const Copyright = () => {
  return <p className="copyright">&copy; {format(new Date(), 'yyyy')}<br/>Tim Mansell</p>;
};

export default Copyright;
