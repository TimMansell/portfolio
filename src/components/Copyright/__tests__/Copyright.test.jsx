import { it, describe, expect } from 'vitest';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import Copyright from '../Copyright';

describe('Copyright', () => {
  it('should render my component', () => {
    // eslint-disable-next-line
    const wrapper = shallow(<Copyright />);
  });

  it('should match snapshot', () => {
    const snapshot = renderer.create(<Copyright />).toJSON();

    expect(snapshot).toMatchSnapshot();
  });
});
