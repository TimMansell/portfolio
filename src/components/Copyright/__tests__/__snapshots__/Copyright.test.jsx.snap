// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Copyright > should match snapshot 1`] = `
<p
  className="copyright"
>
  Â© 
  2023
  <br />
  Tim Mansell
</p>
`;
